src/pages/SignupHandler/index.js

```diff
@@ -145,7 +145,21 @@ const SignupHandler = props => {
 
   const handleSignupSubmit = () => {
     const { dispatch } = props;
-    dispatch(routerRedux.push('/login'));
+    dispatch({
+      type: 'auth/registerUser',
+      payload: {
+        firstName,
+        lastName,
+        username: firstName.toLowerCase(),
+        password,
+        email,
+      },
+    }).then(resp => {
+      if (resp.message === 'Successfully registered.') {
```

Comment:
Should ideally display a message on the dashboard as "Signed in successfully", "failed to sign in" otherwise. Should also ensure that we do not reach the exceptions page by any means